app/assets/stylesheets/announcements.scss

```diff
@@ -110,3 +117,51 @@
 .details {
   text-align: justify;
 }
+
+.button {
+   border: 1px solid #0a3c59;
+   background: #a1947a;
+   background: -webkit-gradient(linear, left top, left bottom, from(#e0ddbf), to(#a1947a));
```

Comment:
You actually do not need these things (the ones that start with a dash); a Rails gem will take care of that. You can just remove these